change/@ni-nimble-components-30881da7-4a97-48ad-81c0-db79e1c01c98.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Seems like we don't have anything in Chromatic that would help identify regressions in the control sizing. Seems like it would be useful to add chromatic tests per control that would help identify size changes with and without labels.

Reply:
I added a Storybook test each for text field, text area, and number field. The tests stack two instances of the control vertically, one with a label, one without. I also add a dashed border to more easily see the bounds of the control.
![image](https://user-images.githubusercontent.com/7282195/201435590-4076c9c9-4216-4ff3-bb20-e5b1a1ca1774.png)